misc/meeting-notes.md

```diff
@@ -1,3 +1,18 @@
+# 2019-03-06
+
+- parallel test harness runner landed!
+- we reviewed the content for creating a PR for the Grammar-WG to rust-lang/lang-team
+  - [PR created](https://github.com/rust-lang/lang-team/pull/3)
+- @CAD is still working on [the %% PR](https://github.com/rust-lang-nursery/gll/pull/2)
+
+- @qmx is taking unit-testing as his next task
+  We've had a brief discussion on disambiguation, precedence tables as an option to solve this. Another challenge is that GLL gives us parse forests, so we have to figure out a way to pick one of the trees.
+  - @centril raised the point that we'll have to figure out disambiguation soon
+  - @CAD suggested having some sort of "precedence table" for simple cases
+  - @centril reminds us that GLL generates parse forests, not parse trees
+  
```

Comment:
Roughly:

+ Once we have a way to test the grammar with expectations, we can experiment with post-processing the parse forest and remove ambiguities; from that we hope to gain a declarative way of specifying disambiguation in `.rs` files. Eventually that may be converted into the `.lyg` format.